doc/source/user_guide/ug_case_settings.rst

```diff
@@ -436,7 +436,7 @@ zbgc_nml
    "``f_exude_l``", "real", "fraction of exudation to DOC lipids", "1.0"
    "``f_exude_s``", "real", "fraction of exudation to DOC saccharids", "1.0"
    "``grid_o``", "real", "z biology for bottom flux", "5.0"
-   "``grid_oS``", "real", "z salinity for bottom flux", "5.0"
+   "``grid_oS``", "real", "DEPRECATED", ""
```

Comment:
In previous code deprecations we have moved lines like this into comments, e.g. see https://github.com/CICE-Consortium/Icepack/pull/411/files#diff-5e3296389949243ad10cff84e0cf3b0b81329d2c9391920fb4cd5b4367018d90